packages/time/src/types.d.ts

```diff
@@ -205,6 +230,24 @@ export interface TimerRepeater {
   disable: () => void;
 }
 
+/**
+ * TimeMath supports simple arithmetic on typed Time values, enforcing that
+ * values are combined in type-compatible ways. You can add 3 minutes to 3pm,
+ * or 5 minutes to a half hour, but it makes no sense to add 3pm and 5pm.
+ * Subtracting two Timestamps does produce a useful difference.
+ *
+ * The brands prevent you from accidentally combining time values from different
+ * TimerServices. If some chains track time in blocks, while others
+ * follow wall clock time, using the correct brands means you don't have to worry
+ * about timezones or how time is represented on a particular chain. This also
```

Comment:
nit, timezones shouldn't be a factor because they are merely a difference in formatting, and of course all sensible code should be using seconds-since-unix-epoch. But this does help manage the difference between consensus time on chain 1, consensus time on chain 2, and the time on your local computer, all of which might be nominally unix time but can diverge in various interesting ways.

Maybe:

    The brands prevent you from accidentally combining time values from different
    TimerServices. Some chains track time in blocks, others follow wall clock
    time, some do both. Every local computer has its own unique notion of wall
    clock time. Even when these clocks are talking about the same thing (UTC),
    they can all drift in different ways. Using the correct brands lets you be
    precise about which particular source of time you mean, preventing confusion
    or attacks when the clocks diverge. Thus it is an error to e.g. use a time
    you got from chain A to schedule an event on chain B.

Reply:
done